frigate/http.py

```diff
@@ -576,25 +576,44 @@ def create_event(camera_name, custom_label):
             {"success": False, "message": f"{camera_name} is not a valid camera."}, 404
         )
 
-    if not custom_label:
+    if not label:
         return jsonify(
-            {"success": False, "message": f"{custom_label} must be set."}, 404
+            {"success": False, "message": f"{label} must be set."}, 404
         )
 
     Event.insert(
         id=event_id,
-        label=custom_label,
+        label=label,
         camera=camera_name,
         start_time=now,
         end_time=None,
```

Comment:
Trying to think of use cases so just brainstorming here.

Having the separate `end` call could lead to an event that never ends. What do you think about the possibility of specifying a `default_event_length` that will set the event to end that period of time in the future, but still allow overriding from the `end` endpoint?

Also how about having the ability to add an offset to the start of the event as a query param? I imagine in a doorbell situation you'd want the event to capture the 10s before the button press too.

Reply:
> Having the separate `end` call could lead to an event that never ends. What do you think about the possibility of specifying a `default_event_length` that will set the event to end that period of time in the future, but still allow overriding from the `end` endpoint?

That is what I was considering as part of my todo item `Add JSON args for customization of behavior`. I would want it to be an either-or though.

> Also how about having the ability to add an offset to the start of the event as a query param? I imagine in a doorbell situation you'd want the event to capture the 10s before the button press too.

This makes sense but would potentially come with issues, for example a user who doesn't have 24/7 recording enabled could request x amount of time in the past but it would be impossible to give them that time since the previous recording segments have already been discarded.

Reply:
> That is what I was considering as part of my todo item Add JSON args for customization of behavior. I would want it to be an either-or though.

Sounds good, yeah the more customization the better.

> This makes sense but would potentially come with issues, for example a user who doesn't have 24/7 recording enabled could request x amount of time in the past but it would be impossible to give them that time since the previous recording segments have already been discarded.

Yeah thats an interesting one, as we expect that any calls to the `event/create` endpoint are from right now, does frigate have a cache of the most recent video file regardless of whether 24/7 recording is on or not? _If so_ creating an event in that time would mean the deleting of the files wouldn't happen because there is an event? Probably missing some context here as I am not familiar with the codebase at all so just spitting ideas, sorry if they're misguided!